docs/install_guides/installing-openvino-macos.md

```diff
@@ -1,212 +1,153 @@
-
-8. The **Finish** screen indicates that the core components have been installed:
-   ![](../img/openvino-install-macos-05.png)
-
-   Once you click **Finish** to close the installation wizard, a new browser window opens with the document you’re reading now (in case you installed without it) and jumps to the section with the next installation steps.
-
-The core components are now installed. If you received a message that you were missing external software dependencies (list available under Software Requirements at the top of this guide), you need to install them now before continuing to the next section.
+   For simplicity, a symbolic link to the latest installation is also created: `<INSTALL_DIR>/intel/openvino_2022/`.
 
 ## <a name="set-the-environment-variables"></a>Step 2: Configure the Environment
 
-> **NOTE**: If you installed OpenVINO to a non-default installation directory, replace `/opt/intel/` with the directory where you installed the software.
-
-You must update several environment variables before you can compile and run OpenVINO™ applications. Set persistent environment variables as follows, using vi (as below) or your preferred editor in the terminal:
-
-1. Open the `.bash_profile` file in the current user home directory:
-   ```sh
-   vi ~/.bash_profile
-   ```
-
-2. Press the **i** key to switch to insert mode.
-
-3. Add this line to the end of the file:
-   ```sh
-   source /opt/intel/openvino_2022/setupvars.sh
-   ```
-
-   If you didn't choose the default installation option, replace `/opt/intel/openvino_2022` with your directory.
-
-4. Save and close the file: press the **Esc** key, type `:wq` and press the **Enter** key.
+You must update several environment variables before you can compile and run OpenVINO™ applications. Set environment variables as follows:
 
-5. To verify the change, open a new terminal. You will see `[setupvars.sh] OpenVINO environment initialized`.
+```sh
+source /opt/intel/openvino_2021/bin/setupvars.sh
+```
 
-   **Optional:** If you don't want to change your shell profile, you can run the following script to temporarily set your environment variables when working with the OpenVINO* toolkit:
+If you have more than one OpenVINO™ version on your machine, you can easily switch its version by sourcing `setupvars.sh` of your choice.
 
-   ```sh
-   source /opt/intel/openvino_2021/bin/setupvars.sh
-   ```  
+> **NOTE**: You can also run this script every time when you start new terminal session. Open `~/.bashrc` in your favorite editor, and add `source <INSTALL_DIR>/intel/openvino_2022/bin/setupvars.sh`. Next time when you open a terminal, you will see `[setupvars.sh] OpenVINO™ environment initialized`. Changing `.bashrc` is not recommended when you have many OpenVINO™ versions on your machine and want to switch among them, as each may require different setup.
 
 The environment variables are set. Continue to the next section to configure the Model Optimizer.
```

Comment:
`Continue to the next section to configure the Model Optimizer.` - this step is not relevant any more.